syntax = "proto3";

package unibit.test;

service Whoami {
  rpc Whoami(Request) returns (Response) {}
}

message Request {
}

message MapEnvEntry {
  string key = 1;
  string value = 2;
}

message Ip {
  string ip = 1;
}

message Response {
  string server_name = 1;
  string client_conn = 2;
  repeated Ip server_ip = 3;
  repeated MapEnvEntry env = 4;
}
